python/pyspark/ml/tuning.py

```diff
@@ -136,6 +136,8 @@ def __init__(self, estimator=None, estimatorParamMaps=None, evaluator=None, numF
             self, "evaluator",
             "evaluator used to select hyper-parameters that maximize the cross-validated metric")
         #: param for number of folds for cross validation
+        self._setDefault(seed=0)
```

Comment:
The default value of `seed` is set here: https://github.com/apache/spark/blob/master/python/pyspark/ml/param/shared.py#L365. This line is not necessary.